layouts/default.vue

```diff
@@ -11,6 +11,53 @@ export default {
     return {
       title: this.$t('head.title'),
       ...this.$nuxtI18nHead({ addSeoAttributes: true }),
+      meta: [
+        {
+          hid: 'twitter:title',
+          name: 'twitter:title',
+          content: this.$t('head.title'),
+        },
+        {
+          hid: 'twitter:description',
+          name: 'twitter:description',
+          content: this.$t('hero.description'),
+        },
+        {
+          hid: 'twitter:image',
+          name: 'twitter:image',
+          content: require('@/assets/images/witnet_logo.svg'),
+        },
+        {
+          hid: 'twitter:image:alt',
+          name: 'twitter:image:alt',
+          content: this.$t('head.title'),
+        },
+        {
+          hid: 'og:title',
+          property: 'og:title',
+          content: this.$t('head.title'),
+        },
+        {
+          hid: 'og:description',
+          property: 'og:description',
+          content: this.$t('hero.description'),
+        },
+        {
+          hid: 'og:image',
+          property: 'og:image',
+          content: require('@/assets/images/witnet_logo.svg'),
+        },
+        {
+          hid: 'og:image:secure_url',
+          property: 'og:image:secure_url',
+          content: require('@/assets/images/witnet_logo.svg'),
```

Comment:
Open Graph does not support SVG images:
https://indieweb.org/The-Open-Graph-protocol#Does_not_support_SVG_images